### PR TITLE
FE-722 implement university selection

### PR DESCRIPTION
**Pull request by andrey-dresvyannikov**
https://mindojo.atlassian.net/browse/FE-722

### CODE REVIEW

File: src/js/main.coffee [hunk 1]
```diff
@@ -63,13 +63,67 @@ initTutorsSlider = ->
   )
 
 
+######################################################################################################################
+#
+# Course & class selectors initializers
+#
+######################################################################################################################
+_updateClassOptions = ($classSelect, classOptions) ->
```
Comment (nikdojo) [inline src/js/main.coffee]: I think there is no need to add "_" in this case as soon as it's scoped insode a module builder.

File: src/js/main.coffee [hunk 2]
```diff
@@ -63,13 +63,67 @@ initTutorsSlider = ->
...
+  $classSelect.find('option').remove()
+  $.each(classOptions, (i, item) -> $classSelect.append(item))
+
+_onSignupSubmit = (universityId, universityClassId) ->
+  Cookies.set('universityId', universityId)
+  Cookies.set('universityClassId', universityClassId)
+  if window.location.host.indexOf('dev.mindojo.com') is -1
+    window.location.href = 'https://learn.mindojo.com/signup'
+  else
+    window.location.href = 'https://learn.dev.mindojo.com/signup'
```
Comment (nikdojo) [inline src/js/main.coffee]: Looks like it's hardcoded for domains. Why not parametrical?

File: src/js/main.coffee [hunk 3]
```diff
@@ -63,13 +63,67 @@ initTutorsSlider = ->
...
+
+initCourseSelector = ->
+  $universitySelect = $('#signupForm select[name="universityId"]').first()
```
Comment (nikdojo) [inline src/js/main.coffee]: Why `first()`? Only one name == 'universityId' should exist.

File: src/js/main.coffee [hunk 4]
```diff
@@ -63,13 +63,67 @@ initTutorsSlider = ->
...
+  universityApiUrl = 'https://uni-courses-landing.mindojo.local/api/universities'
+  updateClassOptions =
+  $.get(universityApiUrl, (resp, status) ->
+    return unless status is "success"
+
+    classOptions = {}
+    universityOptions = []
+    for univer in resp.universities
+      classOptions[univer.id] = []
+      universityOptions.push("<option value='#{univer.id}'>#{univer.name}</option>")
```
Comment (nikdojo) [inline src/js/main.coffee]: Why not use handlebars then? ;-)

Comment (andrey-dresvyannikov) [inline src/js/main.coffee]: @nikdojo i think that here it really will be slower and more complicated, if i'll have to rerender template from this data each time user changes university. I now store string with options (not list of strings), so now it should be the most simple and fast way.

**Pull request merged**